dash-pipeline/dockerfiles/Dockerfile.vpp

```diff
@@ -0,0 +1,86 @@
+# This Dockerfile builds an image used to compile vpp for dash date-plane app.
```

Comment:
I think a better name for this would be Dockerfile.vpp-bldr to make it clear it's a builder image and not a run-time image. If it is also used at run-time, then ignore the comment.

Reply:
It serves not only to build dpapp, but also to run vpp.